app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,10 @@
+module CommentsHelper
+    def comment_custom_path(comment,entity,question)
```

Comment:
why need this?
aere we using a separate page?

Reply:
to handle requests